app/components/UserValidator/index.tsx

```diff
@@ -127,6 +132,17 @@ const Validator = (props: Props) => {
             facingMode: 'environment',
           }}
         />
+        <h3>Nylig scannede ABA-IDer</h3>
+        <Flex column justifyContent="center" gap={5}>
+          {scanResults.map(({ message, count }) => (
+            <span key={message + count}>
+              {message} (x{count})
+            </span>
+          ))}
+          {scanResults.length === 0 && (
+            <span>De siste ABA-IDene du scanner vises her.</span>
```

Comment:
```suggestion
            <span>De siste ABA-IDene du scanner vises her</span>
```
Not sure if I dislike the use of periods or not. I'm sort of on the fence

Reply:
Hmm I think I agree that the period should be removed there.. The thing I know that I dislike is that the responses that are shown directly from the backend contain periods, which doesn't really fit in so I'll see if I can find a way to handle that neatly